src/taipy/cli/cli.py

```diff
@@ -15,7 +15,7 @@
 from ._argparse import _ArgumentParser
 
 
-class _CLI:
+class CLI:
```

Comment:
I would keep the '_' on the class name. Otherwise, we consider it to be exposed to the users, and it needs to be documented. That also means we cannot update it without breaking changes.

Reply:
Oh, I thought we was thinking of exposing it to users. Let's discuss it later.